woof-code/rootfs-skeleton/usr/local/petget/installpkg.sh

```diff
@@ -335,6 +345,40 @@ case $DLPKG_BASE in
  ;;
 esac
 
+
+if [ "$USRMERGE" != "" ]; then
+
+   lastdir="$(pwd)"
+
+   cd ${PKG_INSTALL_DIR}/
+
+   #Fix file position for usrmerge FHS
+       
+   mkdir -p usr 2>/dev/null
+
+   [ -d bin ] && mv -f bin usr/
+   [ -d sbin ] && mv -f sbin usr/
+   [ -d lib ] && mv -f lib usr/
+   [ -d lib64 ] && mv -f lib64 usr/
+   [ -d lib32 ] && mv -f lib32 usr/
+   [ -d libx32 ] && mv -f libx32 usr/
```

Comment:
Is there any distro or pet that has /lib64, /lib32 and /libx32? 
All cases I have seen with these these dirs have them in /usr. No?

Reply:
Its just future proofing feature. I modified PPM with future proofing and improved package management reliability in mind.